pyrogram/methods/chats/update_chat_notifications.py

```diff
@@ -0,0 +1,75 @@
+from typing import Union
+from datetime import datetime, timedelta
+
+from pyrogram.raw.functions.account import UpdateNotifySettings
+from pyrogram.raw.types import InputNotifyPeer, InputPeerNotifySettings
+from pyrogram.scaffold import Scaffold
+
+
+class UpdateChatNotifications(Scaffold):
+    async def update_chat_notifications(
+        self,
+        chat_id: Union[int, str],
+        show_previews: bool = None,
+        silent: bool = None,
+        mute_until: Union[int, datetime, timedelta] = None
```

Comment:
This adds inconsistencies with the rest of the API, which currently only allows unix timestamps to be passed as argument.

Reply:
Now it accepts only unix time